Tests/KituraCORSTests/TestCORSOptions.swift

```diff
@@ -207,6 +207,39 @@ class TestCORSOptions : XCTestCase {
             }, headers: ["Origin" : "http://api.bob.com"])
         }
     }
+    
+    func testSimpleWithSameAsOrigin() {
+        performServerTest(router) { expectation in
```

Comment:
FYI, I like to set variables at the top of my tests like this:

```
let inputValue = "http://api.bob.com"
let expectedValue = inputValue
var resolvedValue: String
```

In this case, when doing comparison values, such as:

```
XCTAssertEqual(resolvedValue, expectedValue, "reason")
```

It just gives a little more context as to what is expected of the test.